backend/database/seeders/RoleSeeder.php

```diff
@@ -0,0 +1,67 @@
+<?php
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use App\Models\Permission;
+use App\Models\Role;
+use Illuminate\Database\Seeder;
+
+class RoleSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $roles = Role::getArrayOfAllRoleNames();
+        foreach ($roles as $key => $role) {
+                Role::factory()->create([
+                    'id' => $key + 1,
+                    'name' => $role,
+                ]);
+        }
+
+        Permission::create(['name' => Permission::UPDATE_USERS])
```

Comment:
Since this class is seeding both roles and permissions, I think it should either be:

1. broken into two different classes: `RoleSeeder` and `PermissionSeeder` with the permission seeding code extracted
2. renamed to `RoleAndPermissionSeeder.php`

Reply:
Totally see where you're coming from, but I don't think its a big deal here.  I'll keep a rename in mind as this seeder evolves though.